docs/source/AdministratorGuide/CommandReference/index.rst

```diff
@@ -1,194 +0,0 @@
-====================================
-Administrator Command Reference
-====================================
-
-In this subsection all the dirac-admin commands available are explained. You can 
-get up-to-date documentation by using the -h switch on any of them. The following command line 
-flags are common to all DIRAC scripts making use of the *parseCommandLine* method of the base *Script* class::
-
-      General options: 
-        -o:  --option=         : Option=value to add 
-        -s:  --section=        : Set base section for relative parsed options 
-        -c:  --cert=           : Use server certificate to connect to Core Services 
-        -d   --debug           : Set debug mode (-dd is extra debug) 
-        -h   --help            : Shows this help 
-
-General information:
-
-.. toctree::
-    :maxdepth: 2
-
-    dirac-admin-service-ports
-    dirac-platform
-
-.. _registry_cmd:
-
-Managing Registry:
```

Comment:
All this grouping is now lost. I think this grouping was useful to have

Reply:
shouldn't this be determined by the name / location of the script and in docs.conf? Did I get the idea correctly that rst files should be created automatically?

Reply:
Did you look at how these commands are now listed?

Reply:
yes, grouping is now lost.